### PR TITLE
[Refactoring] - Added Path tests

### PR DESCRIPTION
**Pull request by mmajerni**

### CODE REVIEW

File: src/test/java/apicurito/tests/steps/verification/PathVerifications.java [hunk 1]
```diff
@@ -45,25 +47,38 @@ public void checkThatPathParameterIsCreatedForPath(String parameter, String path
...
     }
 
-    @Then("^check that path parameter \"([^\"]*)\" has type \"([^\"]*)\" formatted as \"([^\"]*)\"$")
-    public void checkThatPathParameterHasTypeFormattedAs(String parameter, String type, String as) {
-        SelenideElement summaryElement = PathUtils.getPathPageRoot().$(PathElements.PATH_PARAMETERS_SECTION).$$(PathElements.PATH_PARAMETERS_ROW)
-                .filter(matchText(parameter)).first().$(By.className("summary"));
+    @Then("^check that path parameter \"([^\"]*)\" has type \"([^\"]*)\"$")
+    public void checkThatPathParameterHasType(String parameter, String expectedType) {       //NEW
+        PathUtils.openPathTypes(parameter);
+        String type = PathUtils.getPathPageRoot().$(PathElements.PATH_PARAMETERS_SECTION).$$(PathElements.PATH_PARAMETERS_ROW)
```
Comment (mcada) [inline src/test/java/apicurito/tests/steps/verification/PathVerifications.java]: We should add these two rows into separate method with one parameter - the element we are looking for. The usage would be something like: `getTextOfParameter(PROPERTY_TYPE)`, wdyt?

File: src/test/resources/preparedAPIs/paramsAndSecurity.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        },
+        {
+            "basic": [],
+            "api": [],
+            "oauth": [
+                "scope-2"
+            ]
+        }
+    ]
+}
```
Comment (mcada) [inline src/test/resources/preparedAPIs/paramsAndSecurity.json]: eof row missing

File: src/test/java/apicurito/tests/steps/verification/PathVerifications.java (hunk 1 above)
Comment (mmajerni) [reply src/test/java/apicurito/tests/steps/verification/PathVerifications.java]: I agree. I will do it later. I am creating jira for that

File: src/test/resources/preparedAPIs/paramsAndSecurity.json (hunk 2 above)
Comment (mmajerni) [reply src/test/resources/preparedAPIs/paramsAndSecurity.json]: It is generated by apicurito without my changes.

**Pull request merged**